frontend/src/components/TransactionItem.tsx

```diff
@@ -144,7 +144,7 @@ function TransactionItem({ tx }: Props) {
                 </span>
               </p>
             </div>
-            <p className="text-sm md:text-base text-muted-foreground break-all w-full truncate">
+            <p className="text-sm md:text-base text-muted-foreground break-all line-clamp-1">
```

Comment:
I don't think break-all is the behavior we want, since it breaks words inline, regardless if they could be moved to the next line: 

https://tailwindcss.com/docs/word-break

Could we give this a try instead? 

https://tailwindcss.com/docs/overflow-wrap#wrapping-mid-word

Reply:
![image](https://github.com/user-attachments/assets/217bb085-59fa-4914-8bdb-c7b7f4809f95)

sounds good. looks a bit inconsistent where it breaks though. depends solely on the text passed. some text introduced ellipsis earlier and some at the last

Reply:
i used break all so that if continuous texts are passed it breaks at the end. for non continous texts break-all and break-words doesn't make much difference

Reply:
I think the problem is because of the sidebar. If we use `break-word`, it assumes the entire viewport width and cuts it off once that width is exceeded.

But `break-all` cuts it off according to available width and is better in this scenario

<table>

<tr>
<td>
break-word
</td>
<td>
break-all
</td>
</tr>

<tr>
<td>
<video src="https://github.com/user-attachments/assets/d17b5830-1ec5-4902-969b-4a595f5bcaa3" />
</td>
<td>
<video src="https://github.com/user-attachments/assets/5cfb9408-d016-4934-b20c-8f6de4d0c588" />
</td>
</tr>

</table>

Reply:
@reneaaron applied a similar fix to the messageboard widget on the home page using `break-anywhere`